### DISCUSSION TITLE
Remove metadata title delimiter and docusarus title

### DISCUSSION THREAD

**Discussion by elitan** [facebook/docusaurus#7535] [Q&A] [answered]
Is it possible to remove the metadata title delimiter and Docusaurus title from the metadata title?

If I have a page with the title: Database the metadata title is: "Database | My Docs"

What I want is to remove the " | My Docs" part of the metadata title so the metadata title is only "Database".

Is it possible? If yes, how?

**Comment by Josh-Cena** [comment, marked as answer]
Please see #5878. Not possible at the moment, but there's an open feature request

**Comment by slorber** [comment]
Will be possible in v3.8, see https://github.com/facebook/docusaurus/pull/11090
